collectors/python.d.plugin/hpssa/README.md

```diff
@@ -0,0 +1,44 @@
+# hpssa
+
+Module collects controller, logical and physical devices health metrics from HP Smart Storage Arrays.
+
+## Requirements:
+
+- `ssacli` program
+- `sudo` program
+- `netdata` user needs to be able to sudo the `ssacli` program without password
+
+To grab stats it executes: `sudo -n ssacli ctrl all show config detail`
+
+It produces:
+
+1. Controller state and temperature
+2. Cache module state and temperature
+3. Logical drive state
+4. Physical drive state and temperature
```

Comment:
I know it's a little strange, but our linter wants two extra spaces between the period and the start of the text.

So:

```
1.   Controller state and temperature
2.   Cache module state and temperature
3.   Logical drive state
4.   Physical drive state and temperature
```